python/cli/prelude_cli/views/interactive.py

```diff
@@ -54,6 +56,7 @@ class Wizard:
 
     def __init__(self, account):
         # SDK controllers
+        self.account = account
```

Comment:
this property is an outlier here.
it is under the title/comment "SDK controllers" but it is not that.